components/webui/imports/ui/SearchView/SearchResults/SearchResultsTable/index.jsx

```diff
@@ -139,22 +154,24 @@ const SearchResultsTable = ({
                                 <pre className={"search-results-content search-results-message"}>
                                     {result.message}
                                 </pre>
-                                {isExtractIrSupported &&
-                                    <div className={"search-results-file-link"}>
-                                        <FontAwesomeIcon icon={faFileLines}/>
-                                        {" "}
-                                        <a
-                                            className={"search-results-file-link"}
-                                            rel={"noopener noreferrer"}
-                                            target={"_blank"}
-                                            title={"View log event in context"}
-                                            href={`${Meteor.settings.public.LogViewerWebuiUrl
-                                            }?origFileId=${result.orig_file_id}` +
+                                <div className={"search-results-file-link"}>
+                                    <FontAwesomeIcon icon={faFileLines}/>
+                                    {" "}
+                                    <a
+                                        className={"search-results-file-link"}
+                                        rel={"noopener noreferrer"}
+                                        target={"_blank"}
+                                        title={"View log event in context"}
+                                        href={`${Meteor.settings.public.LogViewerWebuiUrl}?` +
+                                            `type=${STREAM_TYPE}` +
```

Comment:
Hopefully the linter doesn't complain about this:
```suggestion
                                        href={Meteor.settings.public.LogViewerWebuiUrl +
                                            `?type=${STREAM_TYPE}` +
```

Reply:
Linter does complain about this, so I will keep my original version